javaparser-core/src/main/javacc/java.jj

```diff
@@ -711,36 +711,46 @@ StubUnit StubUnit():
     ModifierHolder modifier;
     TypeDeclaration tn = null;
     ModuleDeclaration module = null;
+    NodeList<AnnotationExpr> annotations = new NodeList<AnnotationExpr>();
+    AnnotationExpr annotation = null;
 }
 {
     ( LOOKAHEAD(2) ";" )*
-    [ LOOKAHEAD(PackageDeclaration()) pakage = PackageDeclaration() ]
+    /*
+     * Reading leading imports to support the old stub files that contain import declarations
```

Comment:
Is this still a documented, supported feature of the stub file format?  I think it is, in which case this comment should be updated.
If it's not a documented feature of the stub file format, then remove the feature from the code.